apps/condo/domains/resident/tasks/helpers/sendResidentsNoAccountNotifications.js

```diff
@@ -193,16 +196,22 @@ const sendResidentsNoAccountNotificationsForPeriod = async (period, billingConte
 
     logger.info({ msg: 'Billing context proceed', billingContextsCount: billingContexts.length, requestPeriod, billingContextId })
 
+    let totalSuccessCount = 0, totaAttemptsCount = 0, totaProcessedCount = 0
+
     for (const billingContext of billingContexts) {
-        const redisVarName = `${REDIS_LAST_DATE_KEY}-${period}-${billingContext.id}`
+        const redisVarName = `${REDIS_LAST_DATE_KEY}-${requestPeriod}-${billingContext.id}`
 
         /**
          * This represents min value for billingReceipt createdAt to start processing from
          * 1. First of all use resendFromDt from CLI parameter if available, else
          * 2. Use createdAt value from last success script execution stored in Redis, else
          * 3. Use thisMonthStart
          */
-        const lastDt = resendFromDt ? resendFromDt.replace(' ', 'T') : await redisClient.get(redisVarName) || thisMonthStart
+        const storedLastDt = await redisClient.get(redisVarName)
```

Comment:
Why is there an abbreviation?
```suggestion
        const storedLastDate = await redisClient.get(redisVarName)
```